lib/memjs/serializers/non-serialization.js

```diff
@@ -0,0 +1,22 @@
+function serializeValue(value) {
```

Comment:
OK, it seems simpler to have only two _exported_ functions:

```javascript
serialize(value, extras)
```

and 

```javascript
deserialize(value, extras)
```

Some serializers might additional, internal functions, to, e.g., extract the type from extras, while others might not (for example, the "non-serialization" serializer).